### DISCUSSION TITLE
Gallery

### DISCUSSION THREAD

**Discussion by jbcaillau** [control-toolbox/control-toolbox.github.io#28] [Ideas]
Follow up to [this gitlab issue](https://gitlab.inria.fr/ct/gallery/-/issues/42)

Some notebooks / examples of the gallery do not execute anymore on `binder`. See, *e.g.*
[Solarsail implicit](https://ct.gitlabpages.inria.fr/gallery/solarsail/solarsail-simple-version-implicit.html)

At least problem with non-pinned versions of the packages in thé `yaml` environment file for `conda`

Also have problems with (previously running) `conda` environment on a linux station.

![05E38233-06F8-4DB0-B7EB-27E5C5EC2993](https://user-images.githubusercontent.com/62183989/228817919-2075da41-595d-4da5-8b4b-b7fbc3896b64.jpeg)

**Comment by jbcaillau** [comment]
@ocots `conda env export --from-history` please 🙏🏽
> c’est déjà bien merci. essaie stp de m’envoyer aussi un
`conda env export --from-history`
cf. https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#exporting-an-environment-file-across-platforms

**Comment by ocots** [comment]
```bash
⌂3.61 [ocots:~] [base] % conda env export --from-history
name: base
channels:
  - defaults
dependencies:
  - conda
  - python=3.9
  - python.app
  - mamba
  - octave
  - jupyterlab
  - ca-certificates
  - certifi
  - openssl
prefix: /Users/ocots/opt/miniconda3
```

**Comment by ocots** [comment]
```bash
⌂3.40 [ocots:~] [base] 3s % conda activate gallery
⌂9.59 [ocots:~] [gallery] 4s % conda env export --from-history
name: gallery
channels:
  - defaults
dependencies:
  - panel
  - scipy=1.4
  - matplotlib=3.1
  - nutopy=0.3
  - bocop=3.1
  - cmake=3.20
  - gfortran_osx-64=7.5
  - jupyterlab=3.0
  - numba=0.53
  - ipympl=0.7
prefix: /Users/ocots/opt/miniconda3/envs/gallery
```

**Reply by jbcaillau** [reply]
thanks! (didn't see the post yesterday 🙏🏽)
